### PR TITLE
OCLOMRS-198:After creating a new Dictionary the Visibility public should map to view while visibility private should map to None

### PR DESCRIPTION
**Pull request by Michyjones**
# JIRA TICKET NAME:
[OCLOMRS-198: After creating a new Dictionary the Visibility public should map to view while visibility private should map to None](https://issues.openmrs.org/browse/OCLOMRS-198)

# Summary:
This is a fix After creating a new Dictionary the Visibility public should map to view while visibility private should map to None

### CODE REVIEW

File: src/components/dashboard/components/dictionary/common/DictionaryModal.jsx [hunk 1]
```diff
@@ -192,11 +192,11 @@ export class DictionaryModal extends React.Component {
...
                       <option value="" />
-                      {(isEditingDictionary && dictionary.public_access === 'View') &&
-                      <option value="View" selected>Private </option>
+                      {(isEditingDictionary && dictionary.public_access === 'None') &&
+                      <option value="None" selected>Private </option>
                       }
-                      <option value="Edit">Public</option>
-                      <option value="View">Private</option>
+                      <option value="View">Public</option>
+                      <option value="None">Private</option>
```
Comment (alexmochu) [inline src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: The display values 'Private' and 'Public' should match the values 'View' and 'None'

**Comment by dkayiwa** [comment]
@Michyjones do you know why this is still open for close to two weeks?

**Comment by Michyjones** [comment]
Sorry about that I thought I had requested for PR merge. Could you please
merge it if there are no changes need to be made.

**Comment by dkayiwa** [comment]
@Michyjones is that the procedure we follow before merging pull requests?

**Comment by Michyjones** [comment]
No, let me follow the proper procedure.

File: src/components/dashboard/components/dictionary/common/DictionaryModal.jsx [hunk 2]
```diff
@@ -192,11 +192,11 @@ export class DictionaryModal extends React.Component {
...
                       <option value="" />
-                      {(isEditingDictionary && dictionary.public_access === 'View') &&
-                      <option value="View" selected>Private </option>
+                      {(isEditingDictionary && dictionary.public_access === 'None') &&
+                      <option value="None" selected>Private </option>
                       }
-                      <option value="Edit">Public</option>
-                      <option value="View">Private</option>
+                      <option value="View">View</option>
+                      <option value="None">None</option>
```
Comment (dondrzzy) [inline src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: What does `none` mean to the end user? I think using Private and Public sends a clearer message but also put into consideration the review from @alexmochu.

Comment (kmunene) [reply src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: Seek clarity from @alexmochu on the naming here.

Comment (alexmochu) [reply src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: @dondrzzy @kmunene This change was requested by the Product Owner, he wanted the 'Public' and 'Private' options to be changed to 'View' and 'None'.

**Comment by dkayiwa** [comment]
@Michyjones why is this still open, 24 days later?

**Comment by Michyjones** [comment]
I have been waiting for the teammates to approve it and working on the changes. This point going forward I will try and make sure my PRs are reviewed and approved on time

**Comment by dkayiwa** [comment]
How long is on time?

**Comment by Michyjones** [comment]
To have it reviewed and approved within a day or at most 2 days

**Comment by dkayiwa** [comment]
Correct. 👍

**Comment by Michyjones** [comment]
Thank you.

**Pull request merged**